app/models/project_content_item.rb

```diff
@@ -19,4 +19,8 @@ def proxied_url
   end
 
   scope :uncompleted, -> { where(done: false) }
+
+  def self.matching_search(query)
```

Comment:
As far as I understand scopes, which is not very well at all, I think the scope method is equivalent? Assuming this was a suggestion, and not a question, I've gone ahead and changed it to a scope, as that is shorter, and more consistent with the existing scope.